lib-routing/src/link/PathFindingLinkWidget.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { DiagramEngine } from '@projectstorm/react-diagrams-core';
+import PathFinding from '../engine/PathFinding';
+import { PathFindingLinkFactory } from './PathFindingLinkFactory';
+import { PathFindingLinkModel } from './PathFindingLinkModel';
+import { DefaultLinkSegmentWidget } from '@projectstorm/react-diagrams-defaults/src/link/DefaultLinkSegmentWidget';
```

Comment:
@dylanvorster I reckon you should use
 ```import { DefaultLinkSegmentWidget } from '@projectstorm/react-diagrams-defaults';``` 
instead of the full path.

Reply:
Fixed that here https://github.com/projectstorm/react-diagrams/pull/379